queue/command_queued.go

```diff
@@ -5,17 +5,28 @@ import (
 	"github.com/micromdm/micromdm/queue/internal/commandqueuedproto"
 )
 
-func MarshalQueuedCommand(udid, uuid string) ([]byte, error) {
+type QueueCommandQueued struct {
+	DeviceUDID  string
+	CommandUUID string
+}
+
+func MarshalQueuedCommand(cq *QueueCommandQueued) ([]byte, error) {
+	if cq == nil {
```

Comment:
This will cause a panic up from the caller. I would prefer 
`panic(you passed a nil thing here)` since it's a programmer error. 
or `return nil, fmt.Errorf(marshalling nil QueueCommmand)`